image/kube-aws-approver/Dockerfile

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Use a multi-stage dockerfile so the binary is built in a dockerfile too https://github.com/coreos/dex/blob/master/Dockerfile